birdhouse/components/stac-browser/default.env

```diff
@@ -10,11 +10,20 @@
 # is a fork of the official stac-browser (https://github.com/radiantearth/stac-browser)
 # which is some commits ahead to provide necessary build argument fixes for redirects by the nginx proxy.
 # version name is slightly tweaked to fulfill 'service-config' JSON schema while leaving an obvious trace of the source
-export STAC_BROWSER_VERSION=3.4.0-dev
+export STAC_BROWSER_VERSION=3.3.5
```

Comment:
downgrade?  possible without side-effect?

Reply:
3.4.0-dev is not an official version, it CRIM's tracking of 3.3.5 + https://github.com/radiantearth/stac-browser/pull/653

It should not be needed anymore, so we can align versions directly, but we still need the CRIM version build to inject the prefix.